toolchain/Makefile

```diff
@@ -1,23 +1,34 @@
 TGT=x-tools
+# don't want shit to be slow and I don't speak makefile so uhhh here's some ai generated bullshit
+# i have looked through it and it does look like I understand what's going on, may god forgive my sins
+JOBS ?= $(shell echo $(MAKEFLAGS) | grep -oP '(?<=-j)\d+')
+JOBS ?= 1
 
 all: make_tarball
 
-.PHONY: crosstool-ng-1.9.1 crosstool_install toolchain_build clean_install \
+.PHONY: crosstool-ng-1.28.0 crosstool_install toolchain_build clean_install \
 	install_replacements do_chmod make_tarball clean
 
-crosstool-ng-1.9.1.tar.bz2:
-	wget http://ymorin.is-a-geek.org/download/crosstool-ng/$@
+crosstool-ng-1.28.0.tar.bz2:
+	wget http://crosstool-ng.org/download/crosstool-ng/$@
 
-crosstool-ng-1.9.1: crosstool-ng-1.9.1.tar.bz2
+crosstool-ng-1.28.0: crosstool-ng-1.28.0.tar.bz2
 	tar -xjf $<
 
-crosstool_install: crosstool-ng-1.9.1
-	cd $< && ./configure --prefix=`pwd`/../$@ && make && make install
+crosstool_install: crosstool-ng-1.28.0
+	cd $< && ./configure --prefix="`pwd`/../$@" && make && make install $(MAKEFLAGS)
+	# the old version recurses it's own makefile so hard it escapes into my history file
+	# also, the configure for that couldn't find bash for some reason
+
 
 toolchain_build: ct-ng.config crosstool_install
 	install -d $@
 	cp ct-ng.config $@/.config
-	cd $@ && ../crosstool_install/bin/ct-ng build
+	cd $@ && ../crosstool_install/bin/ct-ng upgradeconfig && ../crosstool_install/bin/ct-ng build.$(JOBS)
+	# This is slow AS FUCK
+	# How did yall have time for rebuilding this in... ...whatever year it was before google code? On hardware that was probably much older?
+	# Might be a personal issue not gonna lie, maybe I should try Gentoo just for the patience. I've heard that it severely decreases fertility rates, however.
+	# - some newgen skid, 2025
```

Comment:
Haha!

Well, I'm bordering on 50 years old. I grew up with the Commodore 128, where loading a game could easily take 20-30 minutes, and then... end with LOAD ERROR. We're used to it! ;-)

But really, this was a painful part of building Cibyl. Luckily, the toolchain only had to be built once. Today, a Docker image would probably be the way to go.